include/SDL3/SDL_stdinc.h

```diff
@@ -482,6 +501,25 @@ typedef Sint64 SDL_Time;
 #define SDL_PRIX32 "X"
 #endif
 #endif
+/* Specifically for the `long long` -- SDL-specific. */
+#ifndef SDL_PLATFORM_WINDOWS
```

Comment:
Can you flip this to `#ifdef SDL_PLATFORM_WINDOWS`?

Does it make sense to make these lower case? SDL_PRIll?
Should we also add SDL_PRIL for single long?

Reply:
> Can you flip this to `#ifdef SDL_PLATFORM_WINDOWS`?

OK, will do

> Does it make sense to make these lower case? SDL_PRIll?

Not sure really. But it _feels_ a bit more readable?? Should I do it?

> Should we also add SDL_PRIL for single long?

I vote no: it's universal, i.e. C89 already, is it not?
`long long` is the oddball here.

Reply:
> > Does it make sense to make these lower case? SDL_PRIll?
> 
> Not sure really. But it _feels_ a bit more readable?? Should I do it?

Sure, although I'm concerned that people won't be able to tell that it's i ll, instead of III or lll. Maybe @icculus has an opinion?

> > Should we also add SDL_PRIL for single long?
> 
> I vote no: it's universal, i.e. C89 already, is it not? `long long` is the oddball here.

Fair enough.

Reply:
> Sure, although I'm concerned that people won't be able to tell that it's i ll, instead of III or lll.

That came to my mind just after writing my response. So, I suggest leaving those as uppercase for now.